Gems/Multiplayer/Code/Source/NetworkEntity/NetworkEntityManager.cpp

```diff
@@ -493,17 +500,19 @@ namespace Multiplayer
         AzFramework::SpawnAllEntitiesOptionalArgs optionalArgs;
         optionalArgs.m_priority = AzFramework::SpawnablePriority_High;
 
+        AZ::Name netSpawnableName = AZ::Interface<INetworkSpawnableLibrary>::Get()->GetSpawnableNameFromAssetId(netSpawnableAsset.GetId());
+
         // Pre-insertion callback allows us to do network-specific setup for the entities before they are added to the scene
-        optionalArgs.m_preInsertionCallback = [spawnableAssetId = netSpawnableAsset.GetId()](AzFramework::EntitySpawnTicket::Id,
+        optionalArgs.m_preInsertionCallback = [netSpawnableName = netSpawnableName](AzFramework::EntitySpawnTicket::Id,
```

Comment:
Capturing with the same name: `[netSpawnableName = netSpawnableName]` --> `[netSpawnableName]`